packages/react-instantsearch/src/components/SearchBox.js

```diff
@@ -216,7 +224,15 @@ class SearchBox extends Component {
             />
           </symbol>
         </svg>
-        <div role="search" {...cx('wrapper')}>
+        <div
+          role="search"
+          {...cx(
+            'wrapper',
+            this.props.showLoadingIndicator &&
```

Comment:
We can probably create a variable for ` this.props.showLoadingIndicator && this.props.isSearchStalled` since we are using this expression three times. It will be a bit less verbose in the code.

Reply:
And do you have an idea for a name for this variable?

Reply:
Nevermind, it's gonna be isSearchStalled :)